src/Uno.UI/UI/Xaml/DependencyObjectStore.Binder.cs

```diff
@@ -82,7 +82,7 @@ public void SetTemplatedParent(FrameworkElement? templatedParent)
 					);
 				}
 
-				_properties.ApplyTemplatedParent(templatedParent);
+				_properties?.ApplyTemplatedParent(templatedParent);
```

Comment:
`_properties` is now lazy, and force-created if accessed via `Properties` property